lib/core.js

```diff
@@ -555,7 +555,11 @@ Archiver.prototype.append = function(source, data) {
  * @return {this}
  */
 Archiver.prototype.bulk = function(mappings) {
-  console.log('Archiver.bulk() deprecated since 0.21.0');
+  if (typeof process !== 'undefined' && typeof process.emitWarning !== 'undefined') {
+    process.emitWarning('Archiver.bulk() deprecated since 0.21.0', 'DeprecationWarning');
+  } else if (typeof console !== 'undefined' && typeof console.warn !== 'undefined') {
```

Comment:
If you think checking for availability of `console.warn` is overkill, let me know and I'll drop this check.